server/src/test/java/io/crate/protocols/postgres/PostgresWireProtocolTest.java

```diff
@@ -874,6 +874,73 @@ public CompletableFuture<ShardResponse> answer(InvocationOnMock arg0) throws Thr
         readReadyForQueryMessage(channel);
     }
 
+    @Test
+    public void test_send_unsupported_minor_protocol_version() {
```

Comment:
shouldn't we keep only the next test since it tests exactly the same plus the unsupported options?